src/store/mutations.js

```diff
@@ -218,4 +218,19 @@ export default {
 	deleteAlias(state, { account, alias }) {
 		account.aliases.splice(account.aliases.indexOf(alias), 1)
 	},
+	renameMailbox(state, { accountId, mailbox, newName }) {
+		const oldId = mailbox.id
+
+		// Remove the old entry
+		Vue.delete(state.mailboxes, mailbox.databaseId(accountId, oldId))
```

Comment:
`mailbox.databaseId` is an `int`. You're using it as function.

Reply:
oh right, i just copy paste to replace normalizedFolderId